# --DO NOT INCLUDE--
def main():
    pass
def config():
    pass

def FourCC(str):
    return 0

Error=""  # this is the Error message to use in try/except blocks